secure_uninstall/README.md

```diff
@@ -0,0 +1,5 @@
+Secure Uninstall
+================
+
+Ask admin password ('admin_passwd' key from config file)
+before to proceed to module uninstallation
```

Comment:
Agree with @bealdav. In many situations, admin password is know by the customer. (maybe not a good practice of course), but it's the reality. So, OK for me to use admin_passwd, as removing a module is a very rare and important action.